docs/basics/1d_hist.rst

```diff
@@ -266,6 +267,12 @@ The function :func:`make_hist() <plothist.histogramming.make_hist>` returns a a
 
 The examples below show how to create a histogram with a category axis in ``boost_histogram`` and plot it with ``plothist``.
 
+.. note::
```

Comment:
Put maybe this note after the examples? Does the user need t understand this before looking at the example (so keep here)? Or should they see this after they ask themselves why this weird storage (so put after)?